src/components/Timestamp/styled.ts

```diff
@@ -1,9 +1,11 @@
 import { styled } from '@/styles';
 
-const Container = styled('span', {
+const Container = styled('div', {
   typo: { size: 12, height: '17px' },
   color: '$darkGrey',
   whiteSpace: 'nowrap',
+  flexShrink: 0,
+  width: 50,
```

Comment:
we need this to be a set width.

I actually calculated the exact room left after a typical message:
`340-26-8-248-8` = `50`

This is for the sake of the carousel so that it has a consistent timestamp with the other messages, otherwise the timestamp itself is a very arbitrary and variable width depending on the time itself: 12:49 takes up far more room than 1:11

We can't do it based of of 8:40 or something, even if the font changes it will break this:
https://github.com/voiceflow/react-chat/blob/c1b791752450705b685b05af9faeb8ecdb04641c/src/components/SystemResponse/styled.ts#L64-L68